docs/README.md

```diff
@@ -11,8 +11,10 @@ tutorials.
 ### API reference
 
 The API reference is written in the source of itowns, using
-[JSDoc](http://usejsdoc.org/). This is a classic documentation, explaining all
-exposed methods and classes in itowns, and some internal others.
+[JSDoc](http://usejsdoc.org/), along with support for the
+[Markdown](https://daringfireball.net/projects/markdown/) syntax. This is a
```

Comment:
Given that marked.js (used by JSDoc) supports CommonMark and (more importantly I'd say) GitHub-Flavored Markdown, I'm not sure about this link…

Reply:
I don't think it changes a lot in this context, but I concur that the CommonMark cheatsheet is friendlier (https://commonmark.org/help/)

Reply:
I think the main difference with "original markdown.pl" is that CommonMark and GFM both support fenced code blocks (using three backticks as the fence; CommonMark also supports three tildes), with language tag for syntax highlighting.

Reply:
Yes, but in our case it may be better to use `@example` I think, but we never know ;)